pkg/collector/corechecks/servicediscovery/events.go

```diff
@@ -30,6 +30,7 @@ type eventPayload struct {
 	ServiceName                string   `json:"service_name"`
 	GeneratedServiceName       string   `json:"generated_service_name"`
 	GeneratedServiceNameSource string   `json:"generated_service_name_source,omitempty"`
+	ContainerServiceName       string   `json:"container_service_name"`
```

Comment:
should the source tag be added as well as a new field?

Reply:
Added in 3aaed1d765eb991528e8c0d3f42278a085aaa3a4